app/common/AreaFilterSelect.jsx

```diff
@@ -29,12 +29,17 @@ class AreaFilterSelect extends React.Component {
     const polygonSelectorComponent = () => {
       if (this.state.selectedLayer !== null) {
         const placeholderName = `Select ${this.state.selectedLayer.label}`;
+        const sortedValues = this.props.filterDimensions[this.state.selectedLayer.value].values.sort((a,b) => {
```

Comment:
Question: Is this actually comparing the combined alphabetic and numeric values of the NTA's (BK0101 < BK0102) ?

Reply:
yessir

Reply:
NOOOIIIICE!